### PR TITLE
Translation-ja: Add product-owner/ja/05-recap-takeaways

### PR DESCRIPTION
**Pull request by ystk**
This is the 5th PR for #407 .

### CODE REVIEW

File: product-owner/ja/05-recap-takeaways-article-ja.asciidoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+まずは、労りましょう。中間管理職は大変です。
+
+私たちが話した主なメリットのいくつかについても説明していきます。
+私たちは、InnerSourceがボトルネックを取り除くのにどのように役立つかについて、さまざまな方法を話しました。
+また、中間管理職になることで、より多くの責任とより多くの統制力を得る方法について話しました。
+
+また、同僚とより多くコラボレーションすることで、より少ないリソースでより多くの成果を達成でき、その結果、こうした冗長性に実際に対処することができます。
+オープンプロセスは、おそらく以前得ていたよりも多くの信用を、あなたの仕事に与えることを意味しています。
+これらの計画プロセスを実行して公開することで、すべてがより明確になるため、より少ない政治的問題に対処すれば良くなります
```
Comment (johtani) [inline product-owner/ja/05-recap-takeaways-article-ja.asciidoc:13]:
```suggestion
これらの計画プロセスを実行して公開することで、すべてがより明確になるため、政治的問題を少なくすることができます。
```

**Review by johtani — COMMENTED** [review]
@ystk , I left a suggestion. Please take a look.

**Comment by ystk** [comment]
I merged the suggested change. Thanks, @johtani .

**Review by johtani — APPROVED** [review]
LGTM!

**Review by lenucksi — APPROVED** [review]
Thanks you very much for creating this @ystk and the review @johtani! I'll merge it with your LGTM. :tada: :rocket:

**Comment by rrrutledge** [comment]
Great work, everyone!

**Pull request merged**